resolvers.js

```diff
@@ -526,6 +542,27 @@ const Resolvers = {
         logger.error(e.message);
         return e.message;
       }
+    },
+
+    async deleteOpticalImage(_, args) {
+      let {datasetId} = args;
+      const payload = jwt.decode(args.jwt, config.jwt.secret);
+      try {
+        logger.info(args);
+        await checkPermissions(datasetId, payload);
+        const url = `http://${config.services.sm_engine_api_host}/v1/datasets/${datasetId}/del-optical-image`;
+        const body = JSON.stringify({
+            id: datasetId
+        });
+        await fetch(url, {
```

Comment:
I'd check the return value here as per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Checking_that_the_fetch_was_successful), since the query may not have been successful even if no exception has been thrown